net-p2p/amule/files/amule-2.3.3-boost-m4.patch

```diff
@@ -0,0 +1,968 @@
+Bug: https://bugs.gentoo.org/907843
+Source: https://github.com/tsuna/boost.m4/raw/master/build-aux/boost.m4
```

Comment:
Can we BDEPEND on sys-devel/boost-m4 (possibly bumping it first) and copy the macro in from there? It'd avoid needing a huge patch.

Reply:
Sure. I didn't know it existed.

> and copy the macro in from there

I think just removing it from the source dir is enough.

Reply:
Done, thanks.